charts/external-dns/templates/deployment.yaml

```diff
@@ -100,12 +100,14 @@ spec:
             {{- if .Values.txtOwnerId }}
             - --txt-owner-id={{ .Values.txtOwnerId }}
             {{- end }}
+            {{- if and .Values.txtPrefix .Values.txtSuffix }}
+            {{- fail (printf "'txtPrefix' and 'txtSuffix' are mutually exclusive") }}
+            {{- end }}
             {{- if .Values.txtPrefix }}
             - --txt-prefix={{ .Values.txtPrefix }}
-            {{- end }}
-            {{- if and (eq .Values.txtPrefix "") (ne .Values.txtSuffix "") }}
+            {{- else if .Values.txtSuffix }}
             - --txt-suffix={{ .Values.txtSuffix }}
-            {{- end }}
+            {{- end}}
```

Comment:
```suggestion
            {{- end }}
```

Reply:
@ivankatliarchuk final change I think, I didn't get the config correct in my comment and it's ended up in here.

Reply:
I should not blindly copy paste ;-)